guides/common/modules/con_provisioning-cloud-instances-on-google-compute-engine.adoc

```diff
@@ -6,7 +6,7 @@
 :CRname: Google Compute Engine
 
 {ProjectName} can interact with Google Compute Engine (GCE), including creating new virtual machines and controlling their power management states.
-Only image-based provisioning is supported for creating GCE hosts.
+You can only use gold images supported by {Team} with {Project} for creating GCE hosts.
```

Comment:
I think that Satellite-specific means that this line should be within:
```
ifdef::satellite[]
...
endif::[]
```

Reply:
Addressed :+1: